core/controller/src/main/scala/org/apache/openwhisk/core/controller/actions/PrimitiveActions.scala

```diff
@@ -662,6 +668,9 @@ protected[actions] trait PrimitiveActions {
 
   protected val controllerActivationConfig =
     loadConfigOrThrow[ControllerActivationConfig](ConfigKeys.controllerActivation)
+
+  protected val disableStoreResultConfig: Boolean = loadConfig[Boolean](ConfigKeys.disableStoreResult).getOrElse(false)
```

Comment:
Same as other config comment - use defaults from application.conf and `loadConfigOrThrow()`